<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateView</class>
 <widget class="QMainWindow" name="CreateView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Clooney Sheet Generation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="sheet_preview">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>340</width>
      <height>440</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="event_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>100</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Event Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="event_name">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>465</y>
      <width>240</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="num_match_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>495</y>
      <width>100</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Num Matches:</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_spacer_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>495</y>
      <width>110</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Spacer Sheet:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fields_filepath">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>515</y>
      <width>220</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="save_config_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>545</y>
      <width>100</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Config to:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="config_filepath">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>540</y>
      <width>220</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="add_spacer_page">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>492</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="num_matches">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>488</y>
      <width>85</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="save_fields_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>100</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Fields to:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sheets_filepath">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>565</y>
      <width>220</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="save_sheets_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>100</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Sheets to:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fields_fp_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>515</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="config_fp_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>540</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sheets_fp_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>565</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="fields_tree">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>380</width>
      <height>530</height>
     </rect>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="add_field_button">
    <property name="geometry">
     <rect>
      <x>635</x>
      <y>555</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Add Field</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_field_button_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>555</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Do something</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_field_button_3">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>555</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Save Files</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
